### PR TITLE
Feature/eb 6.0 php7.2

### PR DESCRIPTION
**Pull request by thijskh**

### CODE REVIEW

File: roles/engineblock/tasks/install-branch.yml [hunk 1]
```diff
@@ -37,3 +37,4 @@
   notify:
     - "restart httpd"
     - "restart php-fpm"
```
Comment (quartje) [inline roles/engineblock/tasks/install-branch.yml]: We don't need "restart php-fpm" anymore. The app/console commands are also php72 only

**Comment by thijskh** [comment]
I was planning to remove that everywhere when we also remove installing php5 from the playbooks. Not sure which way is better.

**Comment by quartje** [comment]
It doesn't hurt either ...

**Comment by MKodde** [comment]
**Observation:**
this PR adds support for the changes introduced in: https://github.com/OpenConext/OpenConext-engineblock/pull/800 :+1:

**Pull request merged**